packages/thumbprint-react/components/Alert/index.module.scss

```diff
@@ -0,0 +1,66 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+.tpAlert {
+    display: flex;
+    font-size: $tp-font__body__2__size;
+    line-height: $tp-font__body__2__line-height;
+    text-align: left;
+    padding: 12px $tp-space__3;
```

Comment:
Reason the 12px is non-standard?

Reply:
It appears to be 12px to make the height of the entire alert 44px...